.github/workflows/BuildPR.yml

```
@@ -0,0 +1,25 @@
+name: PullRequest
+
+on: [pull_request]
+
+jobs:
+  buildGitx:
```

```suggestion
  build-mgscopebar:
```
Follows the github naming convention, since this will also show up in the branch protection and checks.